### DISCUSSION TITLE
Custom layout that don't view title

### DISCUSSION THREAD

**Discussion by githubsqltoolsse** [McShelby/hugo-theme-relearn#497] [Q&A]
I use platyps to generate documentation for a powershell module. It creates the title by it self. So I don't want the page title to be written, which cause a double title lines.

I can't figure how to do this. Any one that can point me in the right direction.

**Comment by McShelby** [comment]
How does PlaytPS generate the FrontMatter? Can you post an example output of PlaytPS and the commandline you've used?
